models/public/hbonet-0.25/hbonet-0.25.md

```diff
@@ -0,0 +1,62 @@
+# hbonet-0.25
+
+## Use Case and High-Level Description
+
+The `hbonet-0.25` model is one of the classification models from https://github.com/d-li14/HBONet with `width_mult=0.25`
```

Comment:
```suggestion
The `hbonet-0.25` model is one of the classification models from [repository](https://github.com/d-li14/HBONet) with `width_mult=0.25`
```
*Hereinafter*

Reply:
Not sure that it's better. As for me, author's name should be visible

Reply:
Fair enough, but for consistency with others models my suggestion is preferable

Reply:
👍 for @eizamaliev, however, you could easily expand the description to include the author name too:

> The `hbonet-0.25` model is one of the classification models described by [HBONet: Harmonious Bottleneck on Two Orthogonal Dimensions](https://arxiv.org/abs/1908.03888) by Duo Li, Aojun Zhou and Anbang Yao, and implemented in Duo Li's [repository](https://github.com/d-li14/HBONet). This model has `width_mult=0.25`.

Reply:
replaced